pkg/koordlet/metricsadvisor/framework/config.go

```diff
@@ -36,6 +36,8 @@ type Config struct {
 	CPICollectorTimeWindow           time.Duration
 	ColdPageCollectorInterval        time.Duration
 	EnablePageCacheCollector         bool
+	CollectPromMetricInterval        time.Duration
```

Comment:
what's the differences with ScrapeInterval

Reply:
removed, use the individual interval for each job